Student-Unity-frontend/src/app/components/navigation-bar/navigation-bar.component.ts

```diff
@@ -18,8 +18,16 @@ export class NavigationBarComponent implements OnInit {
     this.getSignedInUser();
   }
 
-  navigateTo(child: string){
+  navigateTo(event: any, child: string){
     this.router.navigateByUrl("home/" + child);
+    let el = event.target;
```

Comment:
what's going here

Reply:
Fixed this issue using router active for coloring the button blue when the URL changes.